samples/DispatcherSample/Startup.cs

```diff
@@ -17,6 +16,7 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddSingleton<UrlGenerator>();
             services.AddSingleton<RouteValueAddressTable>();
+            services.AddSingleton<IStartupFilter, DispatcherEndpointStartupFilter>();
```

Comment:
You should create an 'AddDispatcher()` method and have that responsible for adding the startup filter

Reply:
Add a new `DispatcherServiceCollectionExtensions` class in the `Microsoft.Extensions.DependencyInjection` namespace